# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/jimmy/mplayer/no-freetype21.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

 Albeit both FT_Get_{First,Next}_Char seem to exist in freetype
 library, calling them, at least from mplayer, results in linkage
 undefined references. FT_Get_Char_Index links fine, but the cost
 is a slower MPlayer (with subtitles) startup. Any bright ideas?

--- ./libvo/font_load_ft.c.orig	2003-01-06 14:23:59.000000000 +0200
+++ ./libvo/font_load_ft.c	2003-01-26 14:41:20.000000000 +0200
@@ -30,7 +30,7 @@
 #include "osd_font.h"
 
 #if (FREETYPE_MAJOR > 2) || (FREETYPE_MAJOR == 2 && FREETYPE_MINOR >= 1)
-#define HAVE_FREETYPE21
+// #define HAVE_FREETYPE21
 #endif
 
 char *get_path(char *filename);
